demo-playground/src/main/java/com/google/android/flexbox/validators/NonNegativeDecimalInputValidator.kt

```diff
@@ -14,21 +14,23 @@
  * limitations under the License.
  */
 
-package com.google.android.flexbox.validators;
+package com.google.android.flexbox.validators
 
-import android.text.TextUtils;
+import android.text.TextUtils
 
 /**
  * Validator for non negative integers.
  */
-public class NonNegativeDecimalInputValidator implements InputValidator {
+class NonNegativeDecimalInputValidator : InputValidator {
 
-    @Override
-    public boolean isValidInput(CharSequence charSequence) {
+    override fun isValidInput(charSequence: CharSequence): Boolean {
```

Comment:
Replace the whole body with `return charSequence.toString().toFloatOrNull() ?: -1f >= 0`